Verify.md

```diff
@@ -0,0 +1,82 @@
+# Verify Contracts
+
+## Notes
+1. Contract validation via this script currently does not work on Windows (truffle-flattener generates invalid flattened .sol files).
+2. Success of the validation cannot be positively confirmed via the script without either checking on etherscan.io itself by or rerunning this script several minutes later to confirm.   (You can also run the -g option with the GUID to confirm the outcome).
+3. At this writing, error messages given by the -g option are useless.  The only way to determine why verification is failing is to run the verification manually on the Etherscan website.
+
+## Setup
+1. In the project root, create an ".env" file (or use the existing one) that contains:
+```
+APIKEY=<API key from [etherscan](https://etherscan.io/apis)>
+```
+
+Alternately you can supply this as an OS environment variable when you run `build.verify`.
+
+2. initialize just one time, required by `build.verify`:
+
+```script
+npm run verify.initialize
+```
+
+3. build just one time or whenever you modify verify.ts, required by `verify.build`:
+
+```script
+npm run verify.build
+```
+
+## Verify
+
+### Verify all of the migrated contracts
```

Comment:
So what does this mean exactly? There are different contract versions on chain, and different versions of arc etc. 
I'd expect "very the migrated contracts" means: take the current versino of arc (from package.json), take the addresses from migration.json, and check if the compiled code of arc corresponded to the deployed bytecode.

Is that what is happening?

Reply:
Also, there is some codce in the migartion script that compares bytecodes, perhaps you can look at that as well?

Reply:
What we are doing is, for each contract whose address appears in migration.json (and therefore is a "migrated contract"), we send the contract's solidity code to etherscan.io which will attempt to "verify" that the given code corresponds to the deployed bytecodes.

I will explain this in the md file.

Reply:
done